pkg/health/server/health.go

```diff
@@ -33,7 +37,9 @@ func (h *getHealthz) getCiliumStatus() (*ciliumModels.StatusResponse, error) {
 // Handle handles GET requests for /healthz .
 func (h *getHealthz) Handle(params GetHealthzParams) middleware.Responder {
 	log.Debug("Handling request for /healthz")
-
+	if _, err := os.Stat(option.Config.SocketPath); os.IsNotExist(err) {
+		return api.Error(GetHealthzFailedCode, fmt.Errorf("%s\nIs the agent running?", option.Config.SocketPath))
```

Comment:
```suggestion
		return api.Error(GetHealthzFailedCode, fmt.Errorf("missing socket: %s does not exist", option.Config.SocketPath))
```
I see that you followed the pattern of the CLI, which I think makes sense there, but here I think it would be better to return a single-line error that just indicates what the problem is (the CLI adds  "Is the agent running?" as a helpful message, but in this context we are running within the agent).

Reply:
@thorn3r Updated. Thanks~